services/catalog/src/simcore_service_catalog/core/background_tasks.py

```diff
@@ -183,22 +197,27 @@ async def _ensure_published_templates_accessible(
         await services_repo.upsert_service_access_rights(missing_services_access_rights)
 
 
-async def sync_registry_task(app: FastAPI) -> None:
+async def _sync_services_task(app: FastAPI) -> None:
     default_product: Final[str] = app.state.default_product_name
     engine: AsyncEngine = app.state.engine
+    first_run = True
```

Comment:
still not sure I understand why you add that cache.
we have 10 catalogs with cache enabled already. does this really bring something?

Reply:
My rationale was a follows: 
- 1.  the call to fill ``services_latests`` happen only when there are changes in the registry
- 2. this will not happen until somebody pushes a new image (typically once every day or two?)
- 3. the first ``services_latest`` will be empty for a while so 
- 4. the background task does syn ``services_latests`` also the first run (i.e. upon start) of its infinite loop